mix.exs

```diff
@@ -50,6 +50,8 @@ defmodule CodeFund.Mixfile do
   # Type `mix help deps` for examples and options.
   defp deps do
     [
+      {:credo, "~> 0.9.0", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
```

Comment:
I added `dialyxir`, which is a quality tool to help with typespec'ing elixir functions. As this repo continues to grow and your business logic expands, I would recommend adding `@spec` declarations to your functions, and `@type` around custom defined objects and structs. Dialyxir is a good tool to help with that effort. see: https://elixir-lang.org/getting-started/typespecs-and-behaviours.html

Reply:
:+1: Typespecs are definitely on our list.